src/PhpSpec/Wrapper/Subject/ExpectationFactory.php

```diff
@@ -115,7 +115,7 @@ private function createNegative($name, $subject, array $arguments = array())
      * @param mixed  $subject
      * @param array  $arguments
      *
-     * @return Expectation
+     * @return Expectation|ThrowExpectation
```

Comment:
You seem to be right but that's a worry - from what I can see all instances of `ThrowExpectation` have a method compatible with `Expectation` which is called as if they're an expectation.

Seems to be a bug - the most straightforward fix would be to make the one interface extend the other. Arguably that would break BC because there's a chance someone will have used `ThrowExpectation` without implementing a `match()` method, but I would expect that code to be non-functional.